### DISCUSSION TITLE
How to extract translation from Markdown to msgtr in the PO file

### DISCUSSION THREAD

**Discussion by rffontenelle** [mondeja/mdpo#287] [Q&A] [answered]
I used md2po and po2md to translate, but then I lost the po file. Now I have the original Markdown file and the translated one, and I would like to update the translation. Is it possible to use md2po or any other tool to extract the translation from the translated Markdown file and fill in the PO file?

**Comment by mondeja** [comment, marked as answer]
Nope as mdpo does not work with a structured AST, it just compares the content of the original Markdown blocks with PO `msgid`s content to do the replacements.

It would be great to have an structured AST+CST CommonMark parser to allow that type of conversions and output the same syntax when creating translated markdown contents but I think that currently there is not such thing available.
